src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,213 @@
+                        )
+                    }
+                />
+                <StyledInput
+                    value={lastName}
+                    maxLength="25"
+                    required
+                    pattern="^[A-Za-zÀ-ÿ ,.'-]+$"
+                    placeholder="Last Name"
+                    onChange={e => setLastName(e.target.value)}
+                    onInput={e => e.target.setCustomValidity('')}
+                    onInvalid={e =>
+                        e.target.setCustomValidity(
+                            'Last names should only contain letters. e.g.Doe'
+                        )
+                    }
+                />
+                <StyledInput
+                    type="email"
+                    value={email}
+                    required
+                    pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,3}$"
+                    placeholder="Email Address"
+                    onChange={e => setEmail(e.target.value)}
+                    onInput={e => e.target.setCustomValidity('')}
+                    onInvalid={e =>
+                        e.target.setCustomValidity(
+                            'Please enter a valid email address. e.g. example@email.com'
+                        )
+                    }
+                />
+            </InstructorSection>
+
+            <SectionHeader>
+                <StyledSectionText>Institution's Info</StyledSectionText>
+            </SectionHeader>
+
+            <InstitutionSection>
+                <StyledInput
+                    value={institutionName}
+                    required
+                    placeholder="Name of Institution"
+                    onChange={e => setInstitutionName(e.target.value)}
+                />
+                <StyledFormSelect
+                    value={institutionType}
+                    defaultText="Institution Type"
+                    choices={institutionTypes}
+                    onChange={e => setInstitutionType(e)}
+                />
+            </InstitutionSection>
+
+            <Checkbox
+                onChange={e => setAgreeToEmail(e.target.value)}
+                label="I agree to receive emails from MongoDB around MongoDB for Academia. After submitting, a 
+                MongoDB representative will reach out within five business days."
+                variant="light"
+            />
+
+            <StyledButton
+                disabled={!canSubmit}
+                type="submit"
+                primary
+                hasArrow={false}
+            >
+                Join MongoDB for Academia
+            </StyledButton>
+        </StyledForm>
+    );
+});
+
+const SuccessState = () => (
+    <SuccessContainer>
+        <SuccessIcon />
+        <H3>Thank you for joining!</H3>
+    </SuccessContainer>
+);
+
+const AcademiaSignUpForm = () => {
+    const [success, setSuccess] = useState(null);
+    if (success) {
+        return <SuccessState />;
+    }
+    return (
+        <>
```

Comment:
```suggestion
```
Since we have just one element here we are returning, we don't need the Fragment `<></>`